Snowflake.Data.Tests/IntegrationTests/SFConnectionIT.cs

```diff
@@ -1735,15 +1743,15 @@ public void TestAsyncLoginTimeout()
 
                     }
                     stopwatch.Stop();
-                    int detla = 10; //in case server time slower.
+                    int delta = 10; // in case server time slower.
 
-                    // Should timeout after 5sec
-                    Assert.GreaterOrEqual(stopwatch.ElapsedMilliseconds, 5 * 1000 - detla);
-                    // But never more than 1 sec (max backoff) after the default timeout
-                    Assert.LessOrEqual(stopwatch.ElapsedMilliseconds, (6) * 1000);
+                    // Should timeout after the minimum possible timeout with jitter
+                    Assert.GreaterOrEqual(stopwatch.ElapsedMilliseconds, 1 * 1000 - delta);
```

Comment:
why 1 * 1000 ? should be timeoutSec * 1000

Reply:
Could you remove the restriction `if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) `from all tests you modified? It should be tested on all operating systems.

Reply:
>why 1 * 1000 ? should be timeoutSec * 1000

Same as above where the test could fail if the jitter was the lowest possible value (1, 0.5, 0.25, 0.125, 0.0625, 0.03125)

If the sleep time is made to be at minimum 1 second for each retry then using timeoutSec should work

> Could you remove the restriction if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) from all tests you modified? It should be tested on all operating systems.

Ok

Reply:
Set maxHttpRetries to 0 and replaced assertion with timeoutSec * 1000

Also removed the Windows only restriction